test/src/main/java/org/apache/accumulo/test/conf/store/ZooBasedConfigIT.java

```diff
@@ -371,24 +360,4 @@ public String toString() {
     }
   }
 
-  private static class TestTicker implements Ticker {
```

Comment:
I don't think this should be removed - a fake timer is necessary to exercise the functionally in the test.

Reply:
Can you file leave this alone until I have a change to look into it deeper?

Reply:
I would at least need to remove the call to cleanup above for it to compile.